app/front/scripts/directives/visualizations/piechart/template.html

```diff
@@ -8,6 +8,7 @@
     state="state"
     format-value="formatValue"
     messages="messages"
-    endpoint="{{ params.babbageApiUrl }}">
+    endpoint="{{ params.babbageApiUrl }}"
+    max-slices="5">
```

Comment:
I guess the process would be to push new version to npm (I'll make you a maintainer)
Then update the dependencies in os-viewer and merge the PR.
You will be able to see the modifications in staging.openspending.org (it auto-deploys there).